interfaces/HO-Portal/src/app/program/program-people/program-people.component.html

```diff
@@ -36,7 +44,7 @@ <h2>{{ 'page.program.program-details.section-people'|translate }}</h2>
     {{ 'page.program.program-people.submitPrivacy'|translate }}
   </confirm-prompt>
 
-  <confirm-prompt [disabled]="!enrolledPeople.length"
+  <confirm-prompt [disabled]="!newEnrolledPeople.length || !btnEnabled"
```

Comment:
I think including the condition of `!newEnrolledPeople.length` into the decision for `btnEnabled` would keep all the logic in 1 place... Instead of here AND in the component-file with `btnEnabled`.